data/tests/modules/etc/dnf/modules.d/httpd.module

```diff
@@ -2,5 +2,6 @@
 stream = 2.4
 version = 1
 profiles = 
-enabled = 1
-locked = 0
\ No newline at end of file
+locked = 0
+name=httpd
```

Comment:
Please could you remove the change. It appears when dnf crashes during the test.

Reply:
I have removed the file changes but added a different commit that changes the ModulePackageContainerTest quite a bit in order to not change the files in repo. I can extract it into a separate PR if you prefer.